install_config/web_console_customization.adoc

```diff
@@ -265,6 +265,20 @@ oauthConfig:
 Relative paths are resolved relative to the master configuration file. You must
 restart the server after changing this configuration.
 
+When there are multiple login providers configured or when the
+link:../install_config/configuring_authentication.html#identity-providers[`*alwaysShowProviderSelection*`]
+option in the *_master-config.yaml_* file is set to *true*, each time a user's
+token to {product-title} expires, the user is presented with this custom page
+before they can proceed with other tasks.
+
+[[custom-login-page-example-usage]]
+=== Example Usage
+
+Custom login pages can be used to create Terms of Service pages. They can also
```

Comment:
Suggest: "... can be used to show Terms of Service information." (to avoid hearing "pages" again).